packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -435,14 +437,26 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl = el;
   };
 
+  private closeItem = async () => {
+    this.isClosing = true;
+    // wait for the animation to complete
+    await timeout(100);
```

Comment:
Let's not do this in the test env.... i think this might be breaking tests

Reply:
ouch! actually I just noticed this was async--we should never do raw async methods in a component. always use a task